dbms/src/Flash/Planner/Plans/PhysicalAggregationConvergent.cpp

```diff
@@ -29,29 +29,15 @@ void PhysicalAggregationConvergent::buildPipelineExecGroup(
     assert(!fine_grained_shuffle.enable());
 
     aggregate_context->initConvergent();
-
-    if (unlikely(aggregate_context->useNullSource()))
-    {
-        group_builder.init(1);
-        group_builder.transform([&](auto & builder) {
-            builder.setSourceOp(std::make_unique<NullSourceOp>(
-                exec_status,
-                aggregate_context->getHeader(),
-                log->identifier()));
-        });
-    }
```

Comment:
Why we need to delete these codes?

Reply:
The logic of `AggContext::useNullSource` is wrapped into `AggContext::getConvergentConcurrency` and `AggContext::readForConvergent`, just to simplify the code.